subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -102,6 +102,10 @@ module=NRF_CLOUD
 module-dep=LOG
 module-str=Log level for nRF Cloud
 module-help=Enables nRF Cloud log messages.
-source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+source "subsys/logging/Kconfig.template.log_config"
+
+config NRF_CLOUD_CLIENT_ID_PREFIX
```

Comment:
I think this could be moved up a bit in the file, perhaps together with `NRF_CLOUD_HOST_NAME` to make it appear together with related configs when using menuconfig / guiconfig

Reply:
Will do.